internal/controller/job/controller.go

```diff
@@ -81,12 +82,24 @@ func (c *controller) ExportJob(ctx context.Context, req *rpc.ExportJobRequest) (
 	return &rpc.ExportJobResponse{Jobs: jobs}, nil
 }
 
-func (c *controller) GetAvailableJobs(ctx context.Context, req *rpc.GetAvailableJobsRequest) (*rpc.GetAvailableJobsResponse, error) {
-	// todo add validation using proto validation extension from Kevin
+func (c *controller) GetJobs(ctx context.Context, req *rpc.GetJobsRequest) (*rpc.GetJobsResponse, error) {
+	var v validate.Validator
+	v.ValidateString("worker_id", req.GetWorkerId()).NotEmpty()
+	v.ValidateString("area_key", req.GetAreaKey()).NotEmpty()
+	v.ValidateInt32("radius_meters", req.GetRadiusMeters()).GreaterThan(0)
+	v.ValidateInt32("limit", req.GetLimit()).LessThan(25)
+
+	if v.Error() != nil {
+		return nil, status.Errorf(codes.InvalidArgument, v.Error().Error())
+	}
 
 	estimatedJobs, err := c.jobService.GetEstimatedJobs(ctx, req.GetAreaKey(), req.GetWorkerId(), req.GetRadiusMeters())
```

Comment:
For consistency, I think we should have the Service Layer's function name match the RPC's name.

```
rpc GetAvailableJobs(GetAvailableJobsRequest) returns (GetAvailableJobsResponse) {}

c.jobService.GetAvailableJobs
```